### ISSUE TITLE
Some shortcuts conflict

### ISSUE THREAD

**Issue by OliverJAsh** [jesseduffield/lazygit#883]
For example, the `R` shortcut for "refresh" is available globally, however in some panels it will not work because other shortcuts override it. E.g. in the branches panel, the same shortcut is used for "rename branch".

It would be good if we avoided such conflicts so that global shortcuts worked across all panels.

**Comment by lordlycastle** [comment]
Nah. They're context aware shortcuts. Like pressing same shortcuts do different things based on different places/context in complex programs like IDE etc.

**Comment by jesseduffield** [comment]
I agree that this would be preferable, but there are only so many keybindings available. I do think perhaps that there should be a way to access global keybindings from any panel though, for example through a menu
